ITISFoundation/osparc-simcore
# Freeze chrigel maurer v0

- upon logout alive keys are set to expire in 1 seco (#1527)
- UI/UX: Thumbnail and loading iframe (#1524)
- [bugfix] Fix for is1499 (#1516)
- Little UI improvements for Monday demo (#1515)
- Minor+Patch upgrades in python packages and system+api testing (#1513)
- Cleanup of orphaned services (#1509)
- WIP: add metrics to count services started/stopped (#1510)
- fix thumbnails (#1514)
- [bug] Is479/cc stability (#1500)
- UI/UX Study Editor (#1505)
- Fix wrong redis key (#1504)
- exportButton does not exist for GUEST users (#1503)
- [maintenance] requirements upgrades (#1502)
- UI/UX: Dashboard Cleanup, Follow-Up I (#1497)
- director: renew cache in the background instead of clearing cache (#1496)
- [maintenance] upgrade pytest in entire repo (#1498)
- documentation enhancement (#1481)
- GC no longer removes projects if user is not GUEST (#1491)
- Fix Postgres migration (#1495)
- UI for service submission (#1309)
- Fix "Save as template" (#1493)
- Fixes + some UI/UX (#1471)
- Fine Grained access rights (#1310)
- Fix garbage collection (#1480)
- [Maintenance] package reqs update and extended invitation lifetime  (#1482)
- Bump tenacity from 6.1.0 to 6.2.0 in /packages/postgres-database (#1478)
- Fixes boot failure in devel mode (#1477)
- [maintenance] Upgrades webserver's requirements (#1444)
- in case of service integration 1.0.0 use inputs.json and outputs.json (#1472)
- API gateway: api-keys in webserver, gateway services and client sdk (#1460)
- Fixes f-string-without-interpolation linter error (#1467)
- Increase openStudy timeout (#1462)
- openStudyLink (#1461)
- Is1304/add internal traefik instance for reverse proxy (#1313)
- Bump coverage from 5.0.3 to 5.1 in /packages/service-library (#1447)
- fix env already shows published port (#1443)
- Maintenance/minio endpoint and qooxdoo compiler logs (#1442)
- Bump jest from 25.2.4 to 25.2.7 in /tests/e2e (#1433)
- Add Light theme (#1422)
- sidecar compatible with integration of services 1.0.0 (#1425)
- Fixes on webserver diagnostics submodule and setup in dev-mode (#1430)
- [maintenance] diagnose long response latency (#1421)
- Fix e2e false positives (#1428)
- Bump pytest-mock from 2.0.0 to 3.0.0 in /packages/service-library (#1423)
- Bump werkzeug from 1.0.0 to 1.0.1 in /packages/service-library (#1424)
- [maintenance] audit e2e npm packages (#1416)
- Make Node outputs more visible (#1405)
- fix issue when next_task_nodes is None (#1411)
- Async Sidecar + maintenance pytest-simcore + docker build-kit/X (#1350)
- Maintenance/upgrade trafaret 2 (#1409)
- Bump ujson from 2.0.2 to 2.0.3 in /packages/service-library (#1403)
- webserver's healthcheck monitors and diagnoses slow callbacks as unhealthy (#1406)
- Fix e2e (#1407)
- Study Browser cleanup (#1393)
- [maintenance] pytest-simcore package (#1394)
- Bump alembic from 1.4.1 to 1.4.2 in /packages/postgres-database (#1398)
- Is1268/public api gateway service (#1298)
- Bump eslint from 4.19.1 to 6.8.0 + Bump babel-eslint from 8.2.6 to 10.1.0 (#1381)
- [maintenance] test requirements (#1386)
- Configure host in different environs (#1389)
- Export group (#1266)
- Bump alembic from 1.0.8 to 1.4.1 in /packages/postgres-database (#1385)
- Bump click from 7.0 to 7.1.1 in /packages/postgres-database (#1384)
- Bump docker from 4.0.2 to 4.2.0 in /packages/postgres-database (#1383)
- Bump tenacity from 6.0.0 to 6.1.0 in /packages/postgres-database (#1382)
- Bump yarl from 1.3.0 to 1.4.2 in /packages/postgres-database (#1371)
- Bump ujson from 2.0.1 to 2.0.2 in /packages/service-library (#1374)
- Bump faker from 4.0.1 to 4.0.2 in /packages/postgres-database (#1372)
- Bump certifi from 2019.6.16 to 2019.11.28 in /packages/postgres-database (#1370)
- Bump coverage from 4.5.1 to 5.0.3 in /packages/postgres-database (#1369)
- [maintenance] upgrade urllib3 and psutil (#1366)
- catch exception on socketio connection (#1365)
- Studies of one node open directly in fullscreen view instead displaying the pipeline (#1316)
- [BUGFIX] Hints + (Tags + Creation info) (#1358)
- [enhancement] Check task exists in db and requirements upgrade (#1361)
- [bugfix] check gather return exceptions (#1360)
- fixes script to migrate comp_tasks table (#1357)
- Bump ujson from 1.35 to 2.0.1 in /packages/service-library (#1352)
- [maintenance] missing sync openapi, upgrades pip (#1354)
- [maintenance] black-formatted packages/services and new common.Makefile (#1342)
- [bugfix] ujson missing dependency in director service (#1353)
- only travis can create releases (#1351)
- add integration-version (#1345)
- prevent crash when no user id is found?? (#1349)
- e2e: Check number of output files (#1343)
- fix path to subscripts
- Fix invalid path
- maintenance: refactor CI scripts and simplify github CI workflows (#1337)
- Bump pytest from 5.3.2 to 5.3.5 in /packages/service-library (#1335)
- Bump trafaret from 2.0.0 to 2.0.2 in /packages/service-library (#1334)
- Bump coverage from 4.5.1 to 5.0.3 in /packages/service-library (#1333)
- Bump coveralls from 1.10.0 to 1.11.1 in /packages/service-library (#1332)
- Bump jsonschema from 3.0.1 to 3.2.0 in /packages/service-library (#1331)
- Bump werkzeug from 0.15.4 to 1.0.0 in /packages/service-library (#1328)
- Bump attrs from 19.1.0 to 19.3.0 in /packages/service-library (#1330)
- increasing logging in sidecar (#1323)
- maintenance:  updates vulnerable requirements and minor doc (#1320)
- Bump pytest-docker from 0.6.1 to 0.7.2 in /packages/service-library (#1327)
- Bump tenacity from 6.0.0 to 6.1.0 in /packages/service-library (#1329)
